File: backend/src/main/java/com/seoulawake/backend/SeoulAwakeApplication.java
package com.seoulawake.backend;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class SeoulAwakeApplication {
	public static void main(String[] args) {
		SpringApplication.run(SeoulAwakeApplication.class, args);
	}
}

File: backend/src/main/kotlin/seoulawake/server/SeoulAwakeApplication.kt
package seoulawake.server

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@EnableJpaAuditing
@SpringBootApplication
class SeoulAwakeApplication

fun main(args: Array<String>) {
  runApplication<SeoulAwakeApplication>(*args)
}
